content/docs/04.5-devx/07-palette-virtual-clusters/05-resize-virtual-clusters.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Resize Palette Virtual Clusters"
+metaTitle: "Resize Palette Virtual Clusters"
+metaDescription: "Learn how to resize Palette Virtual Clusters"
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import InfoBox from 'shared/components/InfoBox';
+import WarningBox from 'shared/components/WarningBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+
+
+# Overview
+
+You can resize virtual clusters from the default size of 4 CPU, 4 GiB Memory, 2 GiB Storage. The size you specify can be greater than the Cluster Group allocation. 
```

Comment:
```suggestion
You can resize virtual clusters from the default size of 4 CPU, 4 GiB Memory, 2 GiB Storage. The size you specify cannot be greater than the system-level quota and the tenant user quota. To learn more about resource quotas, refer to the [resource quota](/devx/resource-quota) documentation.
```

The resource quota page has been updated with rich diagrams and examples. That should help address any confusion.

Reply:
done.

Reply:
added suggestion.